src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5171,4 +5171,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureRefExtensionMethods" xml:space="preserve">
     <value>ref or in extension methods</value>
   </data>
+  <data name="ERR_StackAllocConversionNotPossible" xml:space="preserve">
+    <value>Conversion of a stackalloc expression of type '{0}' to type '{1}' is not possible.</value>
+  </data>
+  <data name="IDS_StackAllocExpression" xml:space="preserve">
+    <value>stackalloc expression of type '{0}'</value>
```

Comment:
`Count` is a `BoundExpression` object. Its display will be something like `stackalloc int[int]`. Should we just return the syntax text in this case?

---
In reply to: [133061220](https://github.com/dotnet/roslyn/pull/21006#discussion_r133061220) [](ancestors = 133061220)